docs/products/flink/howto/connect-opensearch.rst

```diff
@@ -0,0 +1,56 @@
+Create an OpenSearch®-based Apache Flink® table
+===============================================
+
+To build data pipelines, Apache Flink® requires source and target data structures to `be mapped as Flink tables <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/sql/create/#create-table>`_. This functionality can be achieved via the `Aiven console <https://console.aiven.io/>`_ or :doc:`Aiven CLI </docs/tools/cli/service/flink>`.
+
+A Flink table can be defined over an existing or new Aiven for OpenSearch® index to be able to sink streaming data. To define a table over an OpenSearch® index, the index name and column data formats need to be defined, together with the Flink table name to use as reference when building data pipelines.
+
+.. Warning:: 
+
+    Aiven for OpenSearch® can only be used as the **target** of a data pipeline. You'll be able to create jobs that write data to an OpenSearch® index. Reading data from an OpenSearch® index is currently not possible.
+
+
+Create an OpenSearch®-based Apache Flink® table with Aiven Console
+------------------------------------------------------------------
+
+To create an Apache Flink table based on an Aiven for OpenSearch® index via Aiven console:
+
+1. Navigate to the Aiven for Apache Flink® service page, and open the **Jobs and Data** tab.
+
+   .. Warning::
+
+      In order to define Flink's tables an :doc:`existing integration <create-integration>` needs to be available between the Aiven for Flink® service and one or more Aiven for OpenSearch® services.
+
+2. Select the **Data Tables** sub-tab and select the Aiven for OpenSearch® integration to use
+
+3. Select the Aiven for OpenSearch® service and the **index** to be used as target for the data pipeline. If you want to use a new index that does not yet exist, write the index name.
```

Comment:
In other words, if the index doesn't exist but you name it, it will get created? If that's so, I'd maybe put `just write the index name and it will be created`, or (if you think that's too long) `just write the index name`.